### PR TITLE
Plenty 363

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,9190 +1,16 @@
 {
   "name": "findologic",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (TheKeymaster) [inline package-lock.json:4]: This looks more like a downgrade than an upgrade. Here are my local versions:

* NPM: `7.18.1`
* Node: `v14.15.4`

Try to reset these changes and try again installing those.

File: package.json [hunk 2]
```diff
@@ -21,11 +21,11 @@
     "gulp-clean-css": "4.2.0",
     "gulp-concat": "^2.6.1",
     "gulp-minify": "^3.1.0",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.0.0",
     "gulp-sourcemaps": "^2.6.4",
     "gulp-symlink": "^2.1.4",
     "gulp-uglify": "^3.0.2",
-    "node-sass": "^4.14.1",
+    "node-sass": "^6.0.1",
```
Comment (TheKeymaster) [inline package.json:28]: These are two major versions. Are we sure nothing broke?

File: package.json [hunk 3]
```diff
@@ -21,11 +21,11 @@
     "gulp-clean-css": "4.2.0",
     "gulp-concat": "^2.6.1",
     "gulp-minify": "^3.1.0",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.0.0",
```
Comment (TheKeymaster) [inline package.json:24]: Preferably don't use versions that are the first e.g. `5.0.0` or `1.0.0`, etc. If there isn't a `5.1` or at least a `5.0.1` version available, I would prefer to stick with any `4.x` version.

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (RimantasAdeoWeb) [reply package-lock.json:4]: My node got updated apparently. After downgrading Node to `v14.15.4` stuff works again. Reverting the commit

File: resources/js/dist/filters-component-min.js [hunk 4]
```diff
@@ -1,2 +1,2 @@
...
...ete n[e][o]}i[s.default.PARAMETER_ATTRIBUTES]=n,delete i[s.default.PARAMETER_PAGE],document.location.search="?"+$.param(i)},getSelectedFilterValue:function(e){var t=this.getSearchParams();if(!(s.default.PARAMETER_ATTRIBUTES in t))return null;var i=t[s.default.PARAMETER_ATTRIBUTES];return e in i?i[e]:null},getUrlParamValue:function(e){var t=this.getSearchParams();return e in t?t[e]:null},setUrlParamValue:function(e,t){var i=this.getSearchParams();i[e]=t,document.location.search="?"+$.param(i)},setUrlParamValues:function(e){var t=this.getSearchParams();e.forEach(function(e){t[e.key]=e.value}),document.location.search="?"+$.param(t)},getKeyByValue:function(e,t){for(var i in e)if(e.hasOwnProperty(i)&&e[i]===t)return i;return-1},getKeyBySuffix:function(e,t){for(var i in e)if(e.hasOwnProperty(i)&&e[i].endsWith(t))return i;return-1},removeAllAttribsAndRefresh:function(){var e=this.getSearchParams();delete e[s.default.PARAMETER_PAGE],delete e[s.default.PARAMETER_ATTRIBUTES],document.location.search="?"+$.param(e)},getFacetIdInfoMap:function(){var e={};return this.$store.state.itemList.facets.forEach(function(t){e[t.id]=t}),e}}}},{"../constants":14}]},{},[8,10,11,12,13,6,7,5,9,1,3,4,2,15]);
```
Comment (TheKeymaster) [inline resources/js/dist/filters-component-min.js:1]: Have you explicitly rebuilt these files after reverting the last commit?

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/js/dist/filters-component-min.js (hunk 4 above)
Comment (dblaichinger) [reply resources/js/dist/filters-component-min.js:1]: I've executed the command again, no new changes detected. So I guess we are fine?!

Screen Shot 2021-08-25 at 16 08 25

Comment (TheKeymaster) [reply resources/js/dist/filters-component-min.js:1]: Looks fine!

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**